src/commands/flex/plugins/release.ts

```diff
@@ -10,7 +10,7 @@ import { createConfiguration as createConfigurationDocs, release as releaseDocs
  * Creates a Flex Plugin Configuration and releases and sets it to active
  */
 export default class FlexPluginsRelease extends CreateConfiguration {
-  static description = createDescription(releaseDocs.description, true);
+  static description = createDescription(releaseDocs.description, false);
```

Comment:
@ktalebian I noticed that the description of this command includes the line that it needs to run in a plugin directory,  this seems to be line of code to fix it

Reply:
Yes!